### ISSUE TITLE
Can u add a ssd1306 monitor to show the tempture?

### ISSUE THREAD

**Issue by AndyZhuo233** [hokoon/RoastESP32#1]
Some time i don't take my laptop to roast.
So can u add a ssd1306 monitor to show the tempture?
That i can write down the tempture on paper.
Thank you!

**Comment by hokoon** [comment]
Yes I can, but sorry I won't. I'm currently focusing on other project.
